README.md

```diff
@@ -1,69 +1,60 @@
 # Getting Started with Create React App
 >This is an implementation of a connection between a Ruby on Rails back-end and React front-end, and bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+## Built With
 
-In the project directory, you can run:
+- `React`
+- `Redux`
+- `Axios`
+- `NPM`
 
-### `npm start`
+## Prerequisites
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Ruby
+- Ruby on Rails
+- PostgreSQL
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Install
 
-### `npm test`
+- Ruby
+- Ruby on Rails(as a gem)
+- Postgresql database
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Getting Started
 
-### `npm run build`
+To get a local copy of the app, run the following commands in your terminal:
+- `git clone https://github.com/stanleySimeon/hello-react-front-end.git`
+- `cd /hello-react-front-end`
+- `bundle install`
 
```

Comment:
- [OPTIONAL] As it is a [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/connect-frontend-frameworks/hello_world_two_apps.md) for this project, 

  - In the README file for the Rails API, add a link to your React app.
  - When submitting for a review - use a link to the pull request in your Rails API.
  - In the repo for the frontend part - include a PR with changes.
Please ensure you add the link to the backend (API) in the frontend(this readme file).

Reply:
Thanks a lot